Sources/MessagingPushAPN/MessagingPushAPN+PushConfigs.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import UIKit
+
+extension MessagingPushAPN {
+    func setupAutoFetchDeviceToken() {
```

Comment:
I think we can write automated tests for this: 

```swift
// Note: non-working pseudocode 

let app = UIApplication()
app.delegate = ....
// before swizzling, we do not expect CIO SDK to be called
delegate.application(didRegisterForRemoteNotificationsWithDeviceToken:)
expect(cioSdk.registerDeviceToken).not.called()

MessagingPushAPN.configure {}
delegate.application(didRegisterForRemoteNotificationsWithDeviceToken:givenToken)
expect(cioSdk.registerDeviceToken).called(givenToken)
```

Could you try adding automated tests to verify this logic?